egs/librispeech/asr1/run.sh

```diff
@@ -249,7 +249,7 @@ fi
 
 if [ ${stage} -le 5 ]; then
     echo "stage 5: Decoding"
-    nj=32
+    nj=1 #32
```

Comment:
I think it's better to use cpu for the original setup.
How about making a variable `rec_nj` in the beginning of the script, and we can change it in run.sh e.g., `./run.sh --stage 5 --rec_decode 1`